ATTRIBUTIONS.md

```diff
@@ -1,42 +1,223 @@
-- https://github.com/B-Con/crypto-algorithms
+### @types/body-parser
+- TypeScript definitions for body-parser
+- By Santi Albo (https://github.com/santialbo/), VILIC VANE (https://vilic.info), Jonathan Häberle (https://github.com/dreampulse)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/body-parser
+
+-------------------------------------------------------------------------------
+
+### @types/commonmark
+- TypeScript definitions for commonmark.js 0.22.1
+- By Nico Jansen (https://github.com/nicojs)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/commonmark
+
+-------------------------------------------------------------------------------
+
+### @types/express
+- TypeScript definitions for Express 4.x
+- By by Boris Yankov https://github.com/borisyankov/.
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/express
+
+-------------------------------------------------------------------------------
+
+### @types/glob 
+- TypeScript definitions for Glob 5.0.10
+- By vvakame (https://github.com/vvakame)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/glob 
+
+-------------------------------------------------------------------------------
+
+### @types/minimist 
+- TypeScript definitions for minimist 1.1.3
+- By Bart van der Schoor (https://github.com/Bartvds), Necroskillz (https://github.com/Necroskillz)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/minimist
+
+-------------------------------------------------------------------------------
+
+### @types/mkdirp
+- TypeScript definitions for mkdirp 0.3.0
+- By Bart van der Schoor (https://github.com/Bartvds)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/mkdirp
+
+-------------------------------------------------------------------------------
+
+### @types/node
+- TypeScript definitions for Node.js v6.x
+- By Microsoft TypeScript (http://typescriptlang.org), DefinitelyTyped (https://github.com/DefinitelyTyped/DefinitelyTyped)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/node
+
+-------------------------------------------------------------------------------
+
+### @types/request
+- TypeScript definitions for request
+- By Carlos Ballesteros Velasco (https://github.com/soywiz), et. al.
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/request
+
+-------------------------------------------------------------------------------
+
+### @types/tape
+- TypeScript definitions for tape v4.2.2
+- By Bart van der Schoor (https://github.com/Bartvds), Haoqun Jiang (https://github.com/sodatea)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/tape
+
+-------------------------------------------------------------------------------
+
+### @types/ws 
+- TypeScript definitions for ws
+- By Paul Loyd (https://github.com/loyd)
+- [MIT][MIT] License
+- https://www.npmjs.com/package/@types/ws
+
+-------------------------------------------------------------------------------
+
+### @types/codemirror
```

Comment:
Let's just call this `codemirror.d.ts` since it's not part of the `@types` repo. Sorry for all the nitpicking! 🙏